pymatgen/io/cp2k/sets.py

```diff
@@ -0,0 +1,856 @@
+
+    """
+    CP2K input set containing the basic settings for performing geometry optimization. Values are all cp2k
+    defaults, and should be good for most systems of interest.
+    """
+
+    def __init__(
+        self,
+        structure,
+        max_drift=3e-3,
+        max_force=4.5e-3,
+        max_iter=200,
+        project_name="Relax",
+        optimizer="BFGS",
+        override_default_params={},
+        **kwargs
+    ):
+
+        """
+        Args:
+            structure:
+            max_drift: Convergence criterion for the maximum geometry change between the current and the
+                last optimizer iteration. This keyword cannot be repeated and it expects precisely one real.
+                Default value: 3.00000000E-003
+                Default unit: [bohr]
+            max_force (float): Convergence criterion for the maximum force component of the current configuration.
+                This keyword cannot be repeated and it expects precisely one real.
+                Default value: 4.50000000E-004
+                Default unit: [bohr^-1*hartree]
+            max_iter (int): Specifies the maximum number of geometry optimization steps.
+                One step might imply several force evaluations for the CG and LBFGS optimizers.
+                This keyword cannot be repeated and it expects precisely one integer.
+                Default value: 200
+            optimizer (str): Specify which method to use to perform a geometry optimization.
+                This keyword cannot be repeated and it expects precisely one keyword. BFGS is a
+                quasi-newtonian method, and will best for "small" systems near the minimum. LBFGS
+                is a limited memory version that can be used for "large" (>1000 atom) systems when
+                efficiency outweights robustness. CG is more robust, especially when you are far from
+                the minimum, but it slower.
+                Default value: BFGS
+        """
+        super(RelaxSet, self).__init__(structure, **kwargs)
+
+        global_section = Global(project_name=project_name, run_type="GEO_OPT")
+
+        geo_opt_params = [
+            Keyword("TYPE", "MINIMIZATION"),
+            Keyword("MAX_DR", max_drift),
+            Keyword("MAX_FORCE", max_force),
+            Keyword("RMS_DR", 1.5e-3),
+            Keyword("MAX_ITER", max_iter),
+            Keyword("OPTIMIZER", optimizer),
+        ]
+        geo_opt = Section("GEO_OPT", subsections={}, keywords=geo_opt_params)
+
+        if not self.check("MOTION"):
+            self.insert(Section("MOTION", subsections={}))
+        self["MOTION"].insert(geo_opt)
+
+        self.insert(global_section)
+        self.update(override_default_params)
+
+
+# TODO: add defaults
+class CellOptSet(DftSet):
+
+    """
+    CP2K input set containing the basic settings for performing geometry optimization. Values are all cp2k
+    defaults, and should be good for most systems of interest.
+    """
```

Comment:
Can you be a little bit more specific in the docstring here about how the intended use of this set is different from `RelaxSet`?

Reply:
The set isn't different from the RelaxSet. It still sets up a calc to do geometry optimization. The only thing is that there are different parameters that can be set in CP2K compared to VASP so the set should be different. Are you suggesting something like RelaxSet should apply to both CP2K and VASP?